api/src/main/java/fade/mirror/Invokable.java

```diff
@@ -101,8 +107,10 @@ default boolean isInvokableWith(@Nullable Object... arguments) {
      * @param type the type of the parameters.
      * @param <P>  the type of the parameters.
      * @return a parameter stream.
+     * @deprecated use {@link #getParameters(Predicate)} & {@link Filter}s instead.
      */
     @Contract(pure = true)
+    @Deprecated(forRemoval = true)
```

Comment:
Remove @Deprecated method now since the library is still in an immature state. Same all all below.

Reply:
This deprecated method has been removed!